index.html

```diff
@@ -4234,15 +4242,19 @@ <h2>Keywords and text
 </tr>
 </table>
 
-<p>Other keywords may be defined for other purposes. Keywords of
-general interest can be registered with the PNG Registration
-Authority (see 4.9 <a href="#extension-and-registration"><span class=
-"xref">Extension and registration</span></a>). It is also
-permitted to use private unregistered keywords. (Private keywords
-should be reasonably self-explanatory, in order to minimize the
-chance that the same keyword is used for incompatible purposes by
-different people.)</p>
+<p>Other keywords MAY be defined by any application for private or general
+interest.</p>
+
+<p>Keywords SHOULD be .</p>
+<ul>
+  <li>reasonably self-explanatory, since the aim is to let other human users
+  understand what the chunk contains; and</li>
+  <li>chosen to minimize the chance that the same keyword is used for
+  incompatible purposes by different applications.</li>
+</ul>
 
+<p>Keywords of general interest SHOULD be listed in [[png-reg]].</p>
```

Comment:
The keywords are [in png-ext](https://w3c.github.io/PNG-spec/extensions/Overview.html#Keywords), not png-reg

Reply:
Will fix